tests/common/portstat_utilities.py

```diff
@@ -63,7 +65,9 @@ def parse_portstat(content_lines):
         return {}
 
     results = {}
-    for line in content_lines[separation_line_number+1:]:
+    for line in content_lines[separation_line_number+1:reminder_line_number]:
```

Comment:
can we change this to:
`for line in content_lines[separation_line_number+1:reminder_line_number-1]:` ?

there are 3 lines I added to the print, 1st and 3rd are 2 empty lines, 2ns line is the Reminder print, the original error failed at the 1st empty line. In this case we don't need line 69-70 then

Reply:
(Pdb) content_lines[separation_line_number+1:reminder_line_number]
...
u'Ethernet280        X        0    0.00 B/s      0.00%         0         0         0        0   0.00 B/s      0.00%         0         0         0', **u''**]

the last empty line will be ignored

Reply:
I intentionally did not do "[separation_line_number+1:reminder_line_number-1]" to keep it generic for future.
In future release if a Reminder is added without an additional empty line, then  "[separation_line_number+1:reminder_line_number-1]" will miss the last line.
Or in case someone adds an additional empty line also.
So I thought we could parse the output from first line after header till line before "Reminder" and skip any empty EOF lines seen in between

Reply:
ok, that makes sense to me